documentation/modules/ROOT/pages/chart-per-service.adoc

```diff
@@ -116,20 +116,34 @@ Update existing release with the newly added configurations:
 helm upgrade simple ./chart/faq
 ---- 
 
+If you installed the charts locally, you can run the following command:
+
+[.console-input]
+[source,bash,subs="attributes+,+macros"]
+----
+curl -X 'GET' 'http://localhost:8080/ask/cee' -H 'accept: application/json'
```

Comment:
I think it is not necessary to set `-X GET` as it is the default behaviour (at least in Mac)